android/src/main/java/com/swmansion/reanimated/nodes/InvokeNode.java

```diff
@@ -0,0 +1,33 @@
+package com.swmansion.reanimated.nodes;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.swmansion.reanimated.NodesManager;
+import com.swmansion.reanimated.Utils;
+import com.swmansion.reanimated.reflection.ReanimatedAccessor;
+import com.swmansion.reanimated.reflection.ReanimatedReflectionHelper;
+
+public class InvokeNode extends Node implements ConnectedNode {
```

Comment:
Why `InvokeNode` is `ConnectedNode`?

Reply:
I've set it up for future use in `createAnimatedComponent` just like `PropsNode`.
If you connect it:
```diff
-  dispatch('module', 'command', tag, ...args)
+  dispatch('module', 'command', ...args)
```
Haven't tested it yet though

Reply:
So I'd rather revert this change if it's not used neither verified :)

Reply:
verified
1f75bc9